### ISSUE TITLE
Interactive map - Qat unit has a weird character in it? And wrong unit description

### ISSUE THREAD

**Issue by KeileeAnn** [UGS-GIO/geolMapPortal#85]
### On what web page are you requesting a change?

https://geology.utah.gov/apps/intgeomap/?view=scene&scale=16904&zoom=15.1&lat=38.68651&lng=-110.11618&layers=500k%2C100k&tilt=1&heading=1&elev=8598&exag=2.5&base=ustopo

### Describe the change(s) you would like to request.

Two weird things I found here as I was looking for a Qat unit description:
-There is a weird character in the description for the unit where the pin is placed near the top of this screenshot.
-The Qat unit circled in yellow has a unit description for Qea...
![image](https://user-images.githubusercontent.com/112975063/191579054-f1e565ac-949c-40ed-b121-be731ec3dff0.png)

Not sure how easy this is for you guys to fix, and it's not a huge deal, but I thought I'd let you know what I found!

### How Urgent is this?

Low

### Additional notes

Let me know if this screenshot doesn't work and I can show you where these units are...
Thanks a bunch!

**Comment by KeileeAnn** [comment]
oh sorry - the weird character is a y with two dots on top, at the end of the word "wood" so it looks like it says "woody" lol

**Comment by marshallrobinson** [comment]
Thanks for the comment, those are both strange indeed. I'll make sure to get them fixed. 

If you run into any more issues specific to the Geologic Map Portal, it has its own place in Github to track issues like these (I just sent you an invite to be a collaborator): https://github.com/UGS-GIO/geolMapPortal/issues/new/choose

**Comment by marshallrobinson** [comment]
I fixed the y with two dots. Easy peasy...

Fixing the yellow-circled Qat will be a bit more involved to fix, but will still be fixed (thanks for finding it). The Moab 30x60 is the map you see here (because the Moab Quad extends a bit further west beyond the river, overlapping the San Rafael Quad), but the unit description that are popping up in this overlapping zone are from San Rafael Desert. The San Rafael Desert unit description is the more current description, but it looks off because the Moab Quad is showing. The next time the Intermediate Scale layer gets updated, I'll have the Moab Quad moved behind the San Rafael Quad.

**Comment by marshallrobinson** [comment]
@JP-GEO, the next time the 30x60_Quads map gets updates, will you please move the Moab Quad visually behind the San Rafael Quad (making sure the layerID does NOT change - if it does change, we gotta update it in the footprints service in AGOL)

**Comment by KeileeAnn** [comment]
Of course, I should have remembered that! That's interesting that you are putting it behind SRD, I didn't realize that would be something you could do. Grant and I are working on some new geology updates for this quad, and he's been telling us in Mapping that once we publish the new SRD quad, it should be the most up to date source for geology in that area and he doesn't want that section of the Moab quad on the website anymore to confuse our users (this is a ways down the road, though, I think.) He wants to keep the full Moab quad somewhere in archives, but not shown on the Mapping Portal, if that makes sense. Maybe putting it behind SRD will be the solution? Just wanted to give you a heads up on what he's thinking. Thank you for working on this!

**Comment by Mackenzie-Cope** [comment]
When Marshall checked the status of this 5/6/2025, it had been fixed. We don't know exactly when, but we suspect it was done when Jessie was still at UGS. See conversation above.
